readme.md

```diff
@@ -38,10 +38,33 @@ The source object to copy properties from.
 
 #### mapper
 
-Type: `(sourceKey, sourceValue, source) => [targetKey, targetValue, mapperOptions?] | mapObjectSkip`
+Type: `(sourceKey, sourceValue, source, path) => [targetKey, targetValue, mapperOptions?] | mapObjectSkip`
 
 A mapping function.
 
+##### path
+
+Type: `string[]`
+
+When using `{deep: true}`, this is the sequence of keys to reach the current value from the `source`, otherwise it is an empty array.
+
+For arrays, the key is the index of the element being mapped.
+
+Example:
+
```

Comment:
This example feels a bit over-complicated. The intention is to just succinctly show off the feature. I would also format the object keys on separate lines to improve readability (same in the logging - line 65).